packages/gas-fee-controller/CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [4.0.0]
+### Removed
+- **BREAKING:** Remove `isomorphic-fetch` ([#1106](https://github.com/MetaMask/controllers/pull/1106))
```

Comment:
This changelog is missing one thing: https://github.com/MetaMask/core/commit/665942e3595c2879983e4dc78e81ee2ea078786a

Reply:
As a suggestion: https://github.com/MetaMask/core/pull/1108/files#r1110405102